lib/balance-tx/lib/internal/Internal/Cardano/Write/Tx.hs

```diff
@@ -373,6 +374,11 @@ fromRecentEra = \case
     RecentEraConway -> CardanoApi.ConwayEra
     RecentEraBabbage -> CardanoApi.BabbageEra
 
+withRecentEra :: RecentEra era -> (IsRecentEra era => a) -> a
+withRecentEra era a = case era of
+    RecentEraBabbage -> a
+    RecentEraConway -> a
```

Comment:
We actually used to have a function like this in the past, but we deleted it.

Reply:
This implies exporting `IsRecentEra` which makes me uncomfortable as it looks to me a random ad-hoc collection of constraints that are obviously implied for Conway and Babbage but we could as well have forgotten some that are useful for the library user or simply overkill for their use cases.